src/models/Event.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, "please provide an event name!"],
+    },
+    group: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Group",
+      unique: true,
+    },
+    creator: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User",
+      unique: true,
+    },
+    date: {
+      type: Date,
+    },
+    votes: {
+      type: String,
```

Comment:
If we want to track who has voted for an activity (in order to show the user they've already voted and for what, and in order to show the host who has voted, and in order to prevent a user from voting for the same thing multiple times, for example) we may want a `vote` to be an association between an activity and a user. Maybe it could be a model by itself, with `activity` and `user` refs -- what do you think, @msrezaie?

Reply:
Oops! I thought I'd updated that comment. I wrote that thinking we were working with a relational DB. But since we're working with Mongo, it seems like the best we could do would probably be to make votes an array of user refs, right?

Reply:
Yeah this whole idea about activities are kind of not super clear to me as of yet. In my understanding, after a host creates an event members of the group can vote on that event alone. And second, your suggestion of making the votes be referenced to members is great actually. I'll test this out.